pennylane/devices/device_api.py

```diff
@@ -177,6 +177,14 @@ def __repr__(self):
         details = f"({', '.join(details)}) " if details else ""
         return f"<{self.name} device {details}at {hex(id(self))}>"
 
+    def __getattr__(self, key):
+        raise AttributeError(
+            f"{type(self).__name__} has no attribute '{key}'. "
+            " You may be looking for a property or method present in the legacy device interface. "
```

Comment:
Does this result in two spaces between lines?

Reply:
Good catch. I'll fix that.